### DISCUSSION TITLE
Mercator Ikuü SPP04G - no Power, Current or Voltage reported

### DISCUSSION THREAD

**Discussion by kellery82** [Koenkk/zigbee2mqtt#24366] [Devices]
Hi, I've recently purchased and added a Mercator Ikuü SPP04G (reporting as Model: SPP04G) and added successfully to Zigbee2MQTT, however, Power, Current and Voltage are reporting as Null and i'm not sure why (being fairly new to Zigbee2MQTT). I have other Mercator Ikuü devices (2 Gang) and they report these values fine. I found a topic on this issue back on 2022, but thought it was resolved. 

Can anyone help or point me in the right direction.

For reference

![image](https://github.com/user-attachments/assets/2f05b50c-9c18-45da-895e-9691cf54502f)

![image](https://github.com/user-attachments/assets/615ec427-c53c-477b-86c9-4599c934f700)

![image](https://github.com/user-attachments/assets/98545c2c-450d-4588-8650-85f97a8f0a62)

**Comment by kellery82** [comment]
For reference, this is the New Device topic I was referring to back in 2022 which @Koenkk commented on and should mean (at least the way I understand it) it would be available

[https://github.com/Koenkk/zigbee2mqtt/issues/13221](https://github.com/Koenkk/zigbee2mqtt/issues/13221)

I do note though that in that URL , this [https://github.com/Koenkk/zigbee-herdsman-converters/commit/5578e9fcb7486f8aa3194462ec4bfc890418e257](https://github.com/Koenkk/zigbee-herdsman-converters/commit/5578e9fcb7486f8aa3194462ec4bfc890418e257) refers to v20.29.0 of the herdsman converters, but with my version (not edge) I have zigbee-herdsman-converters version 20.21.0 - is this the issue?

**Comment by kellery82** [comment]
I've managed to get the values working, it was missing the Bind information. Adding this to match what the Mercator 2 gang power points had (which have the values) fixed the issue. Does this mean that there are issues with the converter?

![image](https://github.com/user-attachments/assets/20b639c6-8612-4660-88c7-2ed43a5a6025)

![image](https://github.com/user-attachments/assets/9d67e4dc-40a2-4f11-b19f-186f560ea87e)

**Comment by aynema** [comment]
I also have this issue and have had to manually add the bindings. Only seems to be the case with the 4 gang units the 2 gang ones appear fine. Which is a little odd as the 4 gang ones are really just two doubles

**Comment by davidlang42** [comment]
Thank you @kellery82 for figuring this out! This worked for me after I had the same problem:
- Ikuu doubles worked fine
- Ikuu quad did not report power/current/voltage

Given this appears to be happening consistently, should we make this discussion into an issue so the bug can be fixed properly?
